app/api/groups/get_participant_progress.go

```diff
@@ -0,0 +1,279 @@
+package groups
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+	"github.com/France-ioi/AlgoreaBackend/app/structures"
+)
+
+// swagger:model groupParticipantProgressResponseRow
+type groupParticipantProgressResponseRow struct {
+	// required: true
+	ItemID int64 `json:"item_id,string"`
+	// required: true
+	NoScore bool `json:"no_score"`
+	// required: true
+	// enum: Chapter,Task,Course,Skill
+	Type string `json:"type"`
+
+	String structures.ItemString `json:"string"`
+
+	// required: true
+	CurrentUserPermissions *structures.ItemPermissions `json:"permissions"`
```

Comment:
the json should probably be "current_user_permissions" as well.

Reply:
fixed